packages/manager/modules/ng-layout-helpers/package.json

```diff
@@ -29,7 +29,11 @@
   },
   "peerDependencies": {
     "@ovh-ux/ng-ovh-api-wrappers": "^4.0.6",
+    "@ovh-ux/ng-translate-async-loader": "^2.1.0",
     "@ovh-ux/ui-kit": "^4.2.0",
-    "angular": "^1.7.9"
+    "@uirouter/angularjs": "^1.0.22",
```

Comment:
Needed for a component utility ?

Reply:
This is not required and has been removed. Thanks